pkg/monitoring/rules/alerts/vms.go

```diff
@@ -31,11 +31,11 @@ var (
 	vmsAlerts = []promv1.Rule{
 		{
 			Alert: "KubevirtVmHighMemoryUsage",
-			Expr:  intstr.FromString("kubevirt_vm_container_free_memory_bytes_based_on_working_set_bytes < 52428800 or kubevirt_vm_container_free_memory_bytes_based_on_rss < 52428800"),
+			Expr:  intstr.FromString("count(kubevirt_vm_container_free_memory_bytes_based_on_working_set_bytes < 52428800 or kubevirt_vm_container_free_memory_bytes_based_on_rss < 52428800)"),
 			For:   ptr.To(promv1.Duration("1m")),
 			Annotations: map[string]string{
-				"description": fmt.Sprintf("Container {{ $labels.container }} in pod {{ $labels.pod }} in namespace {{ $labels.namespace }} free memory is less than %s and it is close to requested memory", fiftyMB.String()),
-				"summary":     "VM is at risk of being evicted and in serious cases of memory exhaustion being terminated by the runtime.",
+				"description": fmt.Sprintf("There are {{ $value }} VMs where free memory is less than %s and it is close to requested memory", fiftyMB.String()),
+				"summary":     "VMs are at risk of being evicted and in serious cases of memory exhaustion being terminated by the runtime.",
```

Comment:
Thanks @machadovilaca!

my 2cents:
* The term VM is a bit confusing here IMO. Perhaps we should emphasize that we're talking about the virt-launcher pod, not the guest.
* Is there a resolution that the admin / VM owner can see? i.e. using [AdditionalGuestMemoryOverheadRatio](https://github.com/kubevirt/kubevirt/blob/96c9f1180eb8770d58a1f74eb70f0a569d074e8e/staging/src/kubevirt.io/api/core/v1/types.go#L2495-L2501)?
* The JIRA ticket linked in the description is restricted to Red Hat employees only. I think it either has to be open for everyone or removed from the description.

Reply:
> Thanks @machadovilaca!
> 
> my 2cents:
> 
> * The term VM is a bit confusing here IMO. Perhaps we should emphasize that we're talking about the virt-launcher pod, not the guest.

What do you think of keeping the VM term in the summary, but then being specific on the description, `X virt-laucher pods have less than 50Mi free memory ...`

> * Is there a resolution that the admin / VM owner can see? i.e. using [AdditionalGuestMemoryOverheadRatio](https://github.com/kubevirt/kubevirt/blob/96c9f1180eb8770d58a1f74eb70f0a569d074e8e/staging/src/kubevirt.io/api/core/v1/types.go#L2495-L2501)?

this is the runbook that is shared with the alert - https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/KubevirtVmHighMemoryUsage.md